bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/structures/TapoBridgeConfiguration.java

```diff
@@ -46,31 +40,4 @@ public final class TapoBridgeConfiguration {
     public Boolean udpDiscoveryEnabled = false;
     public Integer cloudReconnectIntervalM = CONFIG_CLOUD_FIXED_INTERVAL;
     public Integer discoveryIntervalM = 30;
```

Comment:
Please update the default value of discoveryIntervalM to 60 as it is the default value set in the XML description.
And as the name of the parameter in the XML description is "discoveryInterval", you class member must have the same name. So you should remove the final "M".
And you can rather use int and boolean, rather than Boolean and Integer, at least for cloudDiscovery and discoveryInterval which are real config parameters with always a value.

udpDiscoveryEnabled and cloudReconnectIntervalM are not defined in the XML description, so it is a little surprising to find them as field in the configuration object. I have not checked how they are used by the binding. So I cannot be sure if boolean/int would be more appropriate and if constants should be rather used.

Reply:
This let me think that behaviour of config parameter discoveryInterval was even not tested. Please test that the value you setup in UI for your thing is properly propagated in the binding.

Reply:
You're right thx. Fixed and testet it now

Reply:
Now it really should be pushed. Everything looked clear already.